series-advisor/src/components/Others.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Others = () => (
```

Comment:
`Others` as a component name is a little bit confusing, it's not descrpitive enough.